compose-files/common.yaml

```diff
@@ -33,7 +33,7 @@ services:
         "/bin/stacks-node",
         "start",
         "--config",
-        "/src/stacks-blockchain/Config.toml",
+        "${CONFIG_TOML_TO_USE:-/src/stacks-blockchain/Config.toml}",
```

Comment:
This will always be set to `/src/stacks-blockchain/Config.toml` *unless* manual changes are made to the .env file as-is. It's better to assign a variable and export it in the script vs relying on manual changes.

Reply:
fixed. Thanks.